extensions/adobe/experience/decisioning/frequency-capping-constraints.schema.json

```diff
@@ -25,6 +25,11 @@
             "title": "Frequency capping constraint",
             "description": "Frequency capping constraint define additional rules based on prior offer proposition, either across all audiences or for a particular profile identity.",
             "properties": {
+              "xdm:index": {
+                "title": "Index",
+                "description": "Index of the capping object, this is used as suffix for counter id when we have multi capping objects. Index starts from 1 and default value is 0.",
+                "type": "integer"
```

Comment:
@jshaik-adobe , if you don't specify the minimum and maximum constraints, then it will automatically be set to -2147483648 to 2147483647 and validations would be performed accordingly. Is this the range you are expecting? If not, I would recommend adding the min/max values explicitly.

Reply:
@anandphatak Updated with min/max values